spec/scenarios/invoices_spec.rb

```diff
@@ -620,7 +620,7 @@
         expect(new_invoice.total_amount_cents).to eq(1440) # (1000 + 200) * 1.2
 
         # Create event for Dec 18.
-        create(:event, subscription_id: subscription.id, timestamp: DateTime.new(2022, 12, 18), code: metric.code)
+        create(:event, external_subscription_id: subscription.external_id, timestamp: DateTime.new(2022, 12, 18), code: metric.code)
```

Comment:
Layout/LineLength: Line is too long. [132/120] (https://rubystyle.guide#max-line-length)